packages/behandling-felles/src/util/lagDokumentdata.ts

```diff
@@ -3,11 +3,15 @@ import vedtaksbrevtype from '@fpsak-frontend/kodeverk/src/vedtaksbrevtype';
 
 function lagDokumentdata(aksjonspunktModell) {
   if (aksjonspunktModell.skalUndertrykkeBrev) {
-    return { [dokumentdatatype.VEDTAKSBREV_TYPE]: vedtaksbrevtype.INGEN };
+    return {
+      [dokumentdatatype.VEDTAKSBREV_TYPE]: vedtaksbrevtype.INGEN,
+      [dokumentdatatype.VEDTAKSBREV_MAL]: aksjonspunktModell.vedtaksbrevmaler[vedtaksbrevtype.INGEN],
```

Comment:
Må det ikke være null? DokumentmalType.INGEN finnes ikke i formidling

Reply:
det blir null fordi formidling vil returnere null for denne. Men tenkte å gjøre det slavisk her slik som de andre for å holde logikknivået i frontend på minimal og styre alt fra formidling.